changelogs/unreleased/8403-shubham-pampattiwar

```diff
@@ -0,0 +1 @@
+Add Design for Allowing Instance-Level Resource Status Restore
```

Comment:
I'm not a native speaker but would it better to use "object" instead of instance?  I feel the term "object" is more widely used in the context of k8s.

Reply:
Done, updated the PR